test/unit/dockerUtilities.test.ts

```diff
@@ -618,4 +618,64 @@ suite("Docker Utilities", () => {
         assert.strictEqual(result.error, ContainerDeployment.containerDoesNotExistError);
         execStub.resetBehavior();
     });
+
+    test("sanitizeContainerName: should properly sanitize container names", () => {
+        // Test with valid input
+        let result = dockerUtils.sanitizeContainerName("valid-container");
+        assert.strictEqual(result, "valid-container", "Valid name should remain unchanged");
+
+        // Test with alphanumeric and allowed special characters
+        result = dockerUtils.sanitizeContainerName("test_container.1-2");
+        assert.strictEqual(
+            result,
+            "test_container.1-2",
+            "Name with allowed special chars should remain unchanged",
+        );
+
+        // Test with disallowed special characters
+        result = dockerUtils.sanitizeContainerName("test@container!");
+        assert.strictEqual(result, "testcontainer", "Disallowed special chars should be removed");
+
+        // Test with SQL injection attempt
+        result = dockerUtils.sanitizeContainerName("container';DROP TABLE users;--");
+        assert.strictEqual(
+            result,
+            "containerDROPTABLEusers--",
+            "SQL injection chars should be removed",
+        );

```

Comment:
This isn't getting injected into SQL, but maybe a some of the [`docker remove container`](https://docs.docker.com/reference/cli/docker/container/rm/) commands?